find_most_frequent_number.py

```diff
@@ -0,0 +1,34 @@
+"""
+Se da de la tastatura un numar N, dupa care se citesc N numere intr-o lista. Sa se afle care numar apare de
+cele mai multe ori in lista.
+
+INPUT:
+5
+6 9 10 6 3
+
+OUTPUT:
+6
+
+Complexity:
+    Time: O(n) = (1 + 1 + 1 + 1 + n + 1 + n^2 + 1) = (6 + 2n^2) = n^2
```

Comment:
There's also another solution that is only O(n). If you wanna do that as extra exercise, I recommend using dictionaries.